sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_joined_v1/query.sql

```diff
@@ -1,5 +1,5 @@
 SELECT
-  *
+  * EXCEPT (cls_event.profile_group_id)
```

Comment:
Looks like setting the table isn't allowed in the except so maybe something like this
```suggestion
  cls_main.*,
  cls_event.* EXCEPT (profile_group_id)
```